Xamarin.Forms.Platform.Android/VisualElementTracker.cs

```diff
@@ -305,9 +305,18 @@ void UpdateNativeView(object sender, EventArgs e)
 			if (aview is FormsViewGroup)
 			{
 				var formsViewGroup = (FormsViewGroup)aview;
-				formsViewGroup.SendBatchUpdate((float)(view.AnchorX * _context.ToPixels(view.Width)), (float)(view.AnchorY * _context.ToPixels(view.Height)),
-					(int)(view.IsVisible ? ViewStates.Visible : ViewStates.Invisible), view.IsEnabled, (float)view.Opacity, (float)view.Rotation, (float)view.RotationX, (float)view.RotationY, (float)view.Scale,
-					_context.ToPixels(view.TranslationX), _context.ToPixels(view.TranslationY));
+				formsViewGroup.SendBatchUpdate((float)(view.AnchorX * _context.ToPixels(view.Width)),
+											   (float)(view.AnchorY * _context.ToPixels(view.Height)),
+											   (int)(view.IsVisible ? ViewStates.Visible : ViewStates.Invisible),
+											   view.IsEnabled,
+											   (float)view.Opacity,
+											   (float)view.Rotation,
+											   (float)view.RotationX,
+											   (float)view.RotationY,
+											   (float)view.ScaleX,
+											   (float)view.ScaleY,
+											   _context.ToPixels(view.TranslationX),
+											   _context.ToPixels(view.TranslationY));
 			}
 			else
 			{
```

Comment:
The fast renderers on Android will ignore the independent X and Y scale properties initially; we'd need to update `FormsViewGroup.SendViewBatchUpdate` to handle the ScaleX and ScaleY.
But now that I look at it, we could probably just combine those two methods into a single method which takes a `View`.